### PR TITLE
Bindings command line 

### PR DESCRIPTION
**Pull request by vaikas-google**

### CODE REVIEW

**Comment by vaikas-google** [comment]
Ok, this has now been updated to use the new models. Please take a look.

File: cli/cmd/service_bindings.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+		if err != nil {
+			return err
+		}
+		var si model.ServiceInstance
+		err = json.Unmarshal([]byte(i), &si)
+		if err != nil {
+			return err
+		}
+
+		fmt.Printf("%s -> %s\n\t%+v\n", sb.FromServiceInstanceName, si.Name, sb.Parameters)
```
Comment (bmelville) [inline cli/cmd/service_bindings.go]: Might be good to illustrate here whether the from service instance has been instantiated yet or not. (can be a follow up)

**Comment by bmelville** [comment]
Please sign and squash

File: cli/cmd/service_bindings.go (hunk 1 above)
Comment (vaikas-google) [inline cli/cmd/service_bindings.go]: As in something like
 foo (not instantiated yet) -> mysql

Comment (bmelville) [inline cli/cmd/service_bindings.go]: Yeah something like that, or even just a * next to it or (foo) instead of foo

**Comment by vaikas-google** [comment]
Ok, I did squash and sign:
git commit -s --amend --no-edit

**Comment by bmelville** [comment]
LGTM

**Pull request merged**